src/paperless/settings.py

```diff
@@ -317,6 +319,15 @@ def __get_boolean(key, default="NO"):
 DATE_ORDER = os.getenv("PAPERLESS_DATE_ORDER", "DMY")
 FILENAME_DATE_ORDER = os.getenv("PAPERLESS_FILENAME_DATE_ORDER")
 
+# Transformations applied before filename parsing
+FILENAME_PARSE_TRANSFORMS = []
+_filename_parse_transforms = os.getenv("PAPERLESS_FILENAME_PARSE_TRANSFORMS")
+if _filename_parse_transforms:
+    FILENAME_PARSE_TRANSFORMS = [(
```

Comment:
Rather than a multi-line list comprehension, could you please just use a for loop?  Generally I want to go for readability over code you can fit into one statement.

Reply:
I'll change it so it matches your style preferences.